app/config/AppConfig.scala

```diff
@@ -42,5 +42,7 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   lazy val minutesUntilNextAttemptOnCallbackFailure: Int = config.get[Int]("notifications.minutesUntilRetryOnCallbackFailure")
 
+  lazy val configurableTimeMinutes: Int = config.get[Int]("notifications.configurableTimeMinutes")
```

Comment:
Could this be named more descriptively?

Reply:
<img width="486" alt="Screenshot 2023-06-30 at 16 13 13" src="https://github.com/hmrc/penalties-file-notification-orchestrator/assets/58850837/e9f7297b-1ea4-4899-a039-714c65033f68">
Not sure how better to describe it, should it be named in reference to the job?

Reply:
`configurableTimeMinutes` sounds quite generic as if it was used in a few places, ideally it should be named based on its use case